results/OAEI2025/results.md

```
@@ -0,0 +1,103 @@
+# Evaluation Modalities
+We used the dataset v1.0.0 (https://doi.org/10.5281/zenodo.12731588) and precision, recall and F1-score for evaluation and only evaluated equivalence relationships. If matching systems resulted in either errors or zero identified matches, we considered them as failed. Adhering the [OAEI rules](https://oaei.ontologymatching.org/doc/oaei-rules.2.html), we didn't change any settings for the matching systems. 
+
+## Resources
+- VM with 8 x 2.4 GHz cores, 16GB RAM
+
+## Steps to Reproduce the Results
+- Download the evaluation client [evaluation client](https://nightly.link/dwslab/melt/workflows/java_client_upload/master/evaluation-client.zip) as explained in the [documentation](https://dwslab.github.io/melt/matcher-evaluation/client).
+- Download the 2025 (or any other MELT compatible) matchers and put them in the same folder.
+- Run the command
+
+java -jar matching-eval-client-latest.jar --systems ALIN-Seals.zip logmap-melt-oaei-2021-web-latest.tar.gz logmap-bio-melt-oaei-2021-web-latest.tar.gz logmap-kg-melt-oaei-2021-web-latest.tar.gz logmap-lite-melt-oaei-2021-web-latest.tar.gz lsmatch-2.0-web-latest.tar.gz lsmatch-multilingual-2.0-web-latest.tar.gz matcha.zip MDMapperSeals.zip tim-1.0-web-latest.tar.gz  --track http://oaei.webdatacommons.org/tdrs/ dh 2024all --results oaei2025_all_oaeidh_(date +"%Y-%m-%d_%H-%M-%S") > oaei2025_all_oaeidh_(date +"%Y-%m-%d_%H-%M-%S").log
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)